cleanlab/filter.py

```diff
@@ -20,22 +20,24 @@
 
```

Comment:
Please add this sentence to end of docstring for `confident_joint` arg in `find_label_issues`:
https://github.com/cleanlab/cleanlab/blob/21c1730a80a248732951543bf126b203c903db81/cleanlab/filter.py#L186


If \`multi_label\` is True, confident_joint should instead be a one-vs-rest array with shape \`\`(K, 2, 2)\`\`.